### DISCUSSION TITLE
Connection Refused Error encountering

### DISCUSSION THREAD

**Discussion by divyagupta31** [webdriverio/codemod#71] [Migration Issue]
Hi, while running my framework, i am encountering below error quite frequently:
a BeforeAll hook errored, process exiting: node_modules\@wdio\cucumber-framework\build\index.js:239: connect ECONNREFUSED 127.0.0.1:9515.

What could be the reason for this and what's the resolution?

**Comment by christian-bromann** [comment]
@divyagupta31 seems like Chromedriver crashes or is not set up properly. A reproducible example would help to identify the root cause.
